nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -155,6 +160,47 @@ def followWaypoints(self, poses):
         self.result_future = self.goal_handle.get_result_async()
         return True
 
+    def spin(self, spin_dist=1.57, time_allowance=10):
```

Comment:
Can you add an example python file for these in use? 

Also, might be worth changing the API to `isComplete()` from `isNavComplete()`  since its being used in so many more contexts now (and update the other files that use it)

Reply:
Sure I can add a demo using spin and wait calls. Do you have an idea for a meaningful demo scenario using wait and spin? Otherwise I would make a simple script that uses them (e.g navigate in front of a wall, backup, spin, navigate somewhere else). then again that might just look like a cheap/fake way of demo-ing recoveries.

Reply:
That's fine to me, its more about showing how to use them / get info on feedback than making the demo hyper realistic.